releases/v1.1.5.toml

```diff
@@ -0,0 +1,33 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.1.4"
+
+pre_release = false
+
+preface = """\
+This is the fifth patch release for the `containerd` 1.1 release. This
+release includes several fixes in both the CRI plugin and containerd-shim.
+
+## Containerd
+* Fix a bug that containerd-shim may hang when many exec processes simultaneously
+run in a container.
+* Ignore `modprobe overlay` errors in the systemd unit file. This fixes containerd
+start failures in some containerized environment, e.g. LXC.
```

Comment:
really minor, but we could mention this affected not just LXC but kernels with overlay built-in, and also use cases where overlay isn't being used (and maybe not available on a specialized host OS).  Any of these situations would have a `modprobe` non-zero return code, causing startup to fail.

Reply:
Done. Feel free to correct any grammar error, :p

Reply:
LGTM :)